includes/syslog.php

```diff
@@ -53,7 +53,8 @@ function process_syslog($entry, $update)
     }
 
     $entry['host'] = preg_replace('/^::ffff:/', '', $entry['host']);
-    if ($new_host = Config::get("syslog_xlate.{$entry['host']}")) {
+    $syslog_xlate = Config::get('syslog_xlate');
+    if ($new_host = $syslog_xlate[$entry['host']]) {
```

Comment:
This should like this to avoid errors:

```php
if (! empty($syslog_xlate[$entry['host']])) {
        $entry['host'] = $syslog_xlate[$entry['host']];
```

Reply:
It works for me this way but ok, I changed it